pom.xml

```diff
@@ -167,6 +167,12 @@
             <version>2.2.4</version>
             <scope>test</scope>
         </dependency>
+	<!-- https://mvnrepository.com/artifact/net.snowflake/snowflake-jdbc -->
+	<dependency>
+	    <groupId>net.snowflake</groupId>
+	    <artifactId>snowflake-jdbc</artifactId>
+	    <version>3.12.5</version>
+	</dependency>
         <dependency>
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP-java9ea</artifactId>
```

Comment:
* Correct the group: "areto"
* Correct artifact ID: "snowflake-virtual-schema"
* Start new version numbering (0.1.0) recommended. Increment minor number with features.
* Correct URLs
* ...

Reply:
@redcatbear I suppose you mean line 4 and 5, or no?